Makefile

```diff
@@ -157,10 +157,10 @@ e2e-gpu:
 	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "GPU" -delete-namespace-on-failure=false
 
 e2e-dsa:
-	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "DSA" -delete-namespace-on-failure=false
+	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "DSA" -delete-namespace-on-failure=false -timeout=2400s -ginkgo.timeout=2400s
 
 e2e-iaa:
-	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "IAA" -delete-namespace-on-failure=false
+	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "IAA" -delete-namespace-on-failure=false -timeout=2400s -ginkgo.timeout=2400s
```

Comment:
Does this mean we're increasing test time to 40 min?

Reply:
@bart0sh 
It's a good question. In a wost case, yes.

I am not entirely sure, this is needed and will check if we could drop it (there's some problem within e2e framework ocassionaly, haven't managed to get to the root cause for it yet)